.github/workflows/deploy-to-staging.yml

```diff
@@ -39,14 +39,13 @@ jobs:
 
       - name: Slack Notification
         uses: 8398a7/action-slack@v3
+        if: failure()
         with:
           text: "<!subteam^S06P99RGJDS> <@U07QATMB6SW> <@U07Q99GPBK8> <@U07Q39HFLBG> PYA test suite ${{ job.status }} :sob:"
           status: ${{ job.status }}
           fields: message,commit,author,eventName,ref,workflow,job,took
-          if_mention: failure,cancelled
```

Comment:
this is just for the mentioning & we are not using that field because it doesn’t allow for custom groups, so i’ve removed this.